app/src/main/java/org/mozilla/focus/FocusApplication.kt

```diff
@@ -87,6 +88,8 @@ open class FocusApplication : LocaleAwareApplication(), CoroutineScope {
             components.appStartReasonProvider.registerInAppOnCreate(this)
             components.startupActivityLog.registerInAppOnCreate(this)
 
+            Metrics.searchWidgetInstalled.set(settings.searchWidgetInstalled)
```

Comment:
Can this be moved in [`collectPrefMetrics`](https://github.com/mozilla-mobile/focus-android/blob/1aca30c95aad8b9c15b88375a379770daf8c3d66/app/src/main/java/org/mozilla/focus/telemetry/GleanMetricsService.kt#L93)?

Reply:
Done